context/ifc-context.tsx

```
@@ -1400,10 +1401,152 @@ export function IFCContextProvider({ children }: { children: ReactNode }) {
+                  if (found) {
+                    psetObject = found;
+                    break;
+                  }
+                }
+              }
+            } catch (e) {
+              // ignore
+            }
+          }
+
+          if (psetObject && psetObject.HasProperties) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)